app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkUIView.kt

```diff
@@ -87,19 +82,9 @@ class BookmarkUIView(
 
     fun getSelected(): Set<BookmarkNode> = bookmarkAdapter.selected
 
-    private fun setToolbarColors(foreground: Int, background: Int) {
+    private fun setToolbarColors(@ColorRes foreground: Int, @ColorRes background: Int) {
         val toolbar = activity?.findViewById<Toolbar>(R.id.navigationToolbar)
-        val colorFilter = PorterDuffColorFilter(
-            ContextCompat.getColor(context, foreground), SRC_IN
-        )
-        toolbar?.run {
-            setBackgroundColor(ContextCompat.getColor(context, background))
-            setTitleTextColor(ContextCompat.getColor(context, foreground))
-            themeToolbar(
-                toolbar, foreground,
-                background, colorFilter
-            )
-        }
+        toolbar?.setColors(foreground, background)
```

Comment:
This feels like it could be in a common super class view, shared between `BookmarkUIView` and `HistoryUIView`.